app/experimenter/nimbus-ui/src/components/TableVisualizationRow/index.stories.tsx

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { RouterSlugProvider } from "../../lib/test-utils";
+import { withLinks } from "@storybook/addon-links";
+import { mockExperimentQuery } from "../../lib/mocks";
+import TableVisualizationRow from ".";
+import { mockAnalysis } from "../../lib/visualization/mocks";
+import { DISPLAY_TYPE } from "../../lib/visualization/constants";
+
+const { mock } = mockExperimentQuery("demo-slug");
+
+// TODO: this can use a lot more stories as other tables are built out
```

Comment:
@emtwo as we use this row to display different states, we should add them here. I'm not even sure this one looks that great, but it gets us started.